indico/modules/events/editing/notifications_test.py

```diff
@@ -28,6 +28,9 @@ def test_comment_email_plaintext(snapshot):
     _assert_snapshot(snapshot, 'comment_notification.txt',
                      recipient_name='John Doe',
                      author_name='Jane Doe',
+                     contribution_title='Introduction',
+                     program_code='',
```

Comment:
`contribution=dummy_contribution,`

and add `dummy_contribution` to the test function's arguments